### PR TITLE
Website erstellt

### PR DESCRIPTION
**Pull request by nazligxc**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,336 +1,13 @@
...
-<!-- * * * * * * * * * * and can be replaced.  * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder  * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-<router-outlet></router-outlet>
+<h2>Fika</h2>
+<h1>Zufallsgenerator</h1>
+<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
```
Comment (mcMesa) [inline src/app/app.component.html]: new line after opening tag

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
   templateUrl: './app.component.html',
   styleUrl: './app.component.css'
 })
 export class AppComponent {
-  title = 'zufallsgenerator';
+  random: number | undefined;
+  chosenHistory: Mitarbeiter[] = [];
+  newChosen: any;
+
+  mitarbeiterListe: Mitarbeiter[] = [
```
Comment (mcMesa) [inline src/app/app.component.ts]: maybe extract data set into separate file

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+import { Mitarbeiter } from './mitarbeiter';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrl: './app.component.css'
 })
 export class AppComponent {
-  title = 'zufallsgenerator';
+  random: number | undefined;
```
Comment (mcMesa) [inline src/app/app.component.ts]: random what? be explicit with variable names, e.g. randomColleague

File: src/app/app.component.ts [hunk 4]
```diff
@@ -1,10 +1,41 @@
...
+  chosenHistory: Mitarbeiter[] = [];
```
Comment (mcMesa) [inline src/app/app.component.ts]: how about "lastChosenColleague"

File: src/app/app.component.ts (hunk 2 above)
Comment (nazligxc) [reply src/app/app.component.ts]: resolved

File: src/app/app.component.html (hunk 1 above)
Comment (nazligxc) [reply src/app/app.component.html]: resolved

File: src/app/app.component.ts (hunk 3 above)
Comment (nazligxc) [reply src/app/app.component.ts]: resolved

File: src/app/app.component.ts (hunk 4 above)
Comment (nazligxc) [reply src/app/app.component.ts]: resolved

File: src/app/app.component.html [hunk 5]
```diff
@@ -1,336 +1,13 @@
...
+  Dieser Zufallsgenerator ist dafür da, um den nächsten Fika Speaker 
+  auszuwählen - natürlich ganz zufällig und fair. Klick einfach auf 
+  den Button unten und schau was passiert.
+</p>
+<h3>Nächste Speaker</h3>
+<div *ngIf="newlyChosen != null">
+  <h2>{{ newlyChosen.firstName }} {{ newlyChosen.lastName }}</h2>
+  <img>
+</div>
+<button (click)="onGenerate()">Generieren</button>
```
Comment (mcMesa) [inline src/app/app.component.html]: image
image
layout is jumping, maybe put in a placeholder/whitespace as long no speaker is shown

Comment (mcMesa) [inline src/app/app.component.html]: Generate what exactly? Be more specific., tell your reader what a method does : e.g. selectNextSpeaker()

Comment (nazligxc) [reply src/app/app.component.html]: resolved

Comment (nazligxc) [reply src/app/app.component.html]: resolved

**Pull request merged**